docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,266 @@
+Service endpoint because the Database Service always connects back to the
+cluster over a reverse tunnel.
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db start \
+   --token=/tmp/token \
+   --auth-server=mytenant.teleport.sh:443 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+Note that the `--auth-server` flag must point to your Teleport Cloud tenant
+address.
+
+</ScopedBlock>
+
+</TabItem>
+<TabItem label="Using a config file">
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=teleport.example.com:3080 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=mytenant.teleport.sh:443 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple different services, for example
+  multiple Database Service agents as well as the SSH Service or Application
+  Service.
+
+</Admonition>
+
+Start the database service:
+
+```code
+$ teleport start --config=/path/to/teleport-db.yaml --token=/tmp/token
+```
+
+</TabItem>
+</Tabs>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+
+## Step 3/5. Create a certificate/key pair and create Teleport Oracle Server Wallet
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+Follow the instructions below to generate TLS credentials for your database.
+
+    ```code
+    # Export Teleport's certificate authority and a generate certificate/key pair
+    # for host db.example.com with a 1-year validity period.
+    $ tctl auth sign --format=oracle --host=db.example.com --out=server --ttl=2190h
+    ```
+
+    (!docs/pages/includes/database-access/ttl-note.mdx!)
+
+If [Orapki](https://docs.oracle.com/en/database/oracle/oracle-database/21/dbseg/using-the-orapki-utility-to-manage-pki-elements.html#GUID-0F77F9B8-D425-4A0E-86FC-359D72DAD34F) Oracle tool was found in your local environment the `tctl auth sign --format=oracle --hostdb.example.com --out=server --ttl=2190h` command will produce Oracle Wallet and
+instructions how to configure Oracle TCPS listener with Teleport Oracle Wallet. Otherwise the `tctl auth sign --format=oracle`  will produce `p12` certificate and instructions on how to create an Oracle Wallet on Oracle Server instance.
```

Comment:
```suggestion
instructions how to configure the Oracle TCPS listener with Teleport Oracle Wallet. Otherwise the `tctl auth sign --format=oracle` command  will produce a `p12` certificate and instructions on how to create an Oracle Wallet on your Oracle Server instance.
```